lib/src/models/user/user_clan.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:nyxx/src/client.dart';
+import 'package:nyxx/src/http/cdn/cdn_asset.dart';
+import 'package:nyxx/src/http/route.dart';
+import 'package:nyxx/src/models/guild/guild.dart';
+import 'package:nyxx/src/models/snowflake.dart';
+import 'package:nyxx/src/utils/to_string_helper/to_string_helper.dart';
+
+class UserClan with ToStringHelper {
+  final NyxxRest _client;
```

Comment:
This should be a `UserManager`, not the client - i.e `final UserManager manager;`

Reply:
fair enough, it just seemed weird to me to have a `UserManager` not tied to an actual user.

Reply:
Managers are tied to all the structures they can parse (if they need a manager). It's not really a _must_, it just keeps it consistent with the "primary" SnowflakeEntities that managers handle (i.e User for UserManager) and makes it easy to know where to find the code associated with a specific structure,